.coveragerc

```diff
@@ -2,5 +2,7 @@
 omit =
     */openaerostruct/utils/plot_wing.py
     */openaerostruct/utils/plot_wingbox.py
+    \*tests/\*
```

Comment:
Are the backslashes here necessary?

Reply:
Responding to the comment about the OM docs dependency, we were using sphinx extensions from unsupported OM versions, so we thought it best that for the time being we would transition to sphinx default directives with static outputs.  We are aware this isn't ideal and I think we should discuss the a transition to Jupyter based docs at the maintenance meeting on Monday.

Reply:
As far as the backslashes, that was existing in the `conf.py` file, so I just copied and pasted the regex into the `.coveragerc`.  I'm don't have a great familiarity with regex, so if you think the backslashes should be removed I can get rid of them.

Reply:
My guess is that they should mirror the ones above, e.g. `*/openaerostruct/tests/*` etc. After making the changes, please check that `codecov` is correctly skipping these